modular_zubbers/code/modules/bow_nerf/arrow.dm

```diff
@@ -0,0 +1,110 @@
+/obj/projectile/bullet/arrow
+	damage = 20
```

Comment:
That just makes them useless imo, 45 is what I'd propose. Yes that's a lot of damage but mosins do 60, are also fairly easy to get, and nobody complains about those. For what it's worth mosions have -45 wound_bonus.

Reply:
I see you added +20 damage for Lavaland races, that works. 
Maybe 40 in station hands and 55 in Lavaland hands? With damag - wound_bonus around 10 or so for both, but we'd have to test how much that actually wounds.

Reply:
On the other hand this is readily available to anyone Hmmmmmmm